### ISSUE TITLE
Date format inconsistent

### ISSUE THREAD

**Issue by avotom** [microsoftgraph/microsoft-graph-explorer-v4#968]
**Describe the bug**
When querying applications (https://graph.microsoft.com/v1.0/applications?$select=appId,displayName,keyCredentials,passwordCredentials) the returned date values have different formats, e.g.
"endDateTime": "2299-12-30T23:00:00Z",
"startDateTime": "2020-06-19T06:53:53.24Z"
"endDateTime": "2020-01-11T09:05:32.3052406Z",
"endDateTime": "2019-10-03T06:23:33.322Z",

This makes it hard to parse the dates.

**To Reproduce**
# Send a get request for https://graph.microsoft.com/v1.0/applications?$select=appId,displayName,keyCredentials,passwordCredentials
# Check the date values in keyCredentials or passwordCredentials

**Expected behavior**
I expect consistent format for all date values, best "2019-10-03T06:23:33.322Z"

**Comment by thewahome** [comment]
Thank you for reporting this issue. This appears to be an issue with either your configuration or the service API. This repository is specific to the Graph Explorer web client. Questions about the service API or configuration should be posted to Microsoft Q&A and tagged with one of [microsoft-graph-*](https://docs.microsoft.com/en-us/answers/search.html?c=&includeChildren=&f=&type=question+OR+idea+OR+kbentry+OR+answer+OR+topic+OR+user&redirect=search%2Fsearch&sort=relevance&q=microsoft-graph-*) tags.
